controller/tests/controller_server/addons_server_test.py

```diff
@@ -50,6 +50,10 @@ def test_enable_replication_succeeds(self, storage_agent):
         self.mediator.create_replication.assert_called_once_with(object_internal_id, other_object_internal_id,
                                                                  system_id, copy_type)
 
+    @patch("controller.controller_server.addons_server.get_agent")
+    def test_enable_replication_already_processing(self, storage_agent):
```

Comment:
do we want to cover the other 4 replication methods as well?

Reply:
there is no tests for them..